app/assets/stylesheets/spam2.css

```diff
@@ -36,10 +36,6 @@
 	cursor: pointer;
 }
 
-.alert {
```

Comment:
hey @keshav234156 , inplace of alert what else have you added .
Also can you tell me if you have added something in CSS when a model pop ups, thanks.

Reply:
Alert is there, it is just a z-index for that which I have added when there was another top nav for bulk moderation(previous UI). Now I have moved that nav below the card and there is no need to specify z-index for alert. I have used a modal bootstrap class here.

Reply:
:+1: